<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="2" failures="0" name="news_articles_processor_tests.TestGetToken-20190109233829" skipped="0" tests="2" time="0.008" timestamp="2019-01-09T23:38:29">
	<testcase classname="news_articles_processor_tests.TestGetToken" name="test_get_nouns" time="0.001" timestamp="2019-01-09T23:38:29">
		<error message="too many values to unpack (expected 2)" type="ValueError">
<![CDATA[Traceback (most recent call last):
  File "/home/zia/PycharmProjects/newsrecommender/src/unittest/python/news_articles_processor_tests.py", line 12, in test_get_nouns
    self.assertEqual(get_nouns(['car', 'banana', 'go']), ['car', 'banana'])
  File "/home/zia/PycharmProjects/newsrecommender/src/main/python/news_articles_processor.py", line 60, in get_nouns
    for token, tag_token in tagged:
ValueError: too many values to unpack (expected 2)
]]>		</error>
	</testcase>
	<testcase classname="news_articles_processor_tests.TestGetToken" name="test_get_token" time="0.008" timestamp="2019-01-09T23:38:29">
		<error message="object of type 'function' has no len()" type="TypeError">
<![CDATA[Traceback (most recent call last):
  File "/home/zia/PycharmProjects/newsrecommender/src/unittest/python/news_articles_processor_tests.py", line 9, in test_get_token
    self.assertNotEqual(len(get_tokens('hello world')), 0)
TypeError: object of type 'function' has no len()
]]>		</error>
	</testcase>
	<system-out>
<![CDATA[]]>	</system-out>
	<system-err>
<![CDATA[]]>	</system-err>
</testsuite>
